projects/manager/src/styles.scss

```diff
@@ -0,0 +1,29 @@
+@use '@angular/material' as mat;
+@import 'app-theme';
+
+:root {
+    --geoengine-primary-color: #{mat.get-color-from-palette($geoengine-primary)};
+    --geoengine-accent-color: #{mat.get-color-from-palette($geoengine-accent)};
+    --geoengine-warn-color: #{mat.get-color-from-palette($geoengine-warn)};
+
+    --geoengine-faded-accent-color: #{mat.get-color-from-palette($geoengine-accent, 0.32)};
+
+    --geoengine-foreground-text-color: #{mat.get-color-from-palette(map-get($geoengine-theme, foreground), text)};
+    --geoengine-foreground-secondary-text-color: #{mat.get-color-from-palette(map-get($geoengine-theme, foreground), secondary-text)};
+
+    --geoengine-background-color: #{map-get(map-get($geoengine-theme, background), background)};
+    --geoengine-background-hover-color: #{mat.get-color-from-palette(map-get($geoengine-theme, background), hover)};
+    --geoengine-card-background-color: #{map-get(map-get($geoengine-theme, background), card)};
```

Comment:
all the other dashboards do it exactly this way. So how exactly would you do it differently?

Reply:
I don't get it. The styles.scss is defined in the dasboards and not the core.

Reply:
The core has to access colors. These are stored in CSS variables.
We don't use a core component so nobody uses these variables.

Reply:
> We can import the style file here, right?

which style file do you mean? I need more precise instructions here as I don't know about the workings of the whole styling.

Reply:
For example,
GIS -> can import style using SCSS because it is defined there
CORE -> cannot import style, because it is defined in GIS or EBV Analyzer, etc.

Thus, GIS defines CSS variables and CORE uses them.

But here, nobody uses these variables.

Reply:
well the styles.scss does. So do you want me to delete the app-theme.scss and write it directly in the styles.scss? I am still confused.

Reply:
No, just remove these CSS variables. They should be unused.